apps/prairielearn/src/pages/instructorGradingJob/instructorGradingJob.html.ts

```diff
@@ -150,8 +159,7 @@ export function InstructorGradingJob({
                         </td>
                         <td>
                           Contains the raw output from stdout/stderr for your job. Lines beginning
-                          with
-                          <code>container &gt;</code> are from your container; the rest are
+                          with <code>container&gt;</code> are from your container; the rest are
```

Comment:
As pointed out by @ZacWarham [on Slack](https://prairielearn.slack.com/archives/C0XBTPZMZ/p1748293915735429), there is no space here in the actual log.